dev/com.ibm.ws.microprofile.graphql.authorization/src/com/ibm/ws/microprofile/graphql/authorization/component/AuthorizationInterceptor.java

```diff
@@ -53,12 +55,24 @@ public Object checkAuthorized(InvocationContext ctx) throws Exception {
     @FFDCIgnore({UnauthenticatedException.class})
     private boolean isAuthorized(Method m) {
         try {
-            return RoleMethodAuthUtil.parseMethodSecurity(m, 
-                    AUTH_FILTER.getUserPrincipal(), AUTH_FILTER::isUserInRole);
+            return RoleMethodAuthUtil.parseMethodSecurity(m,
+                    new Supplier<Principal>() {
```

Comment:
Since AUTH_FILTER is static we can just have a single instance of the Supplier stored statically in the class.  Also you should update the Copyright to have `, 2022` added.